db/seeds.rb

```diff
@@ -46,6 +46,13 @@ def random_record_for_org(org, klass)
   partner_key: 'kit'
 )
 
+# ----------------------------------------------------------------------------
+# NDBN Members
+# ----------------------------------------------------------------------------
+NDBNMember.create!(ndbn_member_id: 100, account_name: "Pawnee")
```

Comment:
I think we should make the member id's 5 digits since they are going to be 5 digit numbers.